### PR TITLE
Specify package version in V3 documentation

### LINKED ISSUE #587: Add V3 version to Semaphore V3 documentation commands

### PR DESCRIPTION
**Pull request by cedoor**
## Related Issue

Closes #587

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx [hunk 1]
```diff
@@ -37,19 +37,19 @@ values={[
...
 
 ```bash
-npm install @semaphore-protocol/identity
+npm install @semaphore-protocol/identity@3
 ```
 </TabItem>
 <TabItem value="yarn">
 ```bash
-yarn add @semaphore-protocol/identity
+yarn add @semaphore-protocol/identity@3
```
Comment (vplasencia) [inline apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx]: This worked for you? It installed the version `3`. It worked with `yarn add @semaphore-protocol/identity@3.15.2`.

Comment (cedoor) [reply apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx]: Yes, it should install the latest version. Can you check the `node_modules/@semaphore-protocol/identity/package.json` version?

Comment (vplasencia) [reply apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx]: Yes, it is the right one in node_modules but in the project package json file is `"@semaphore-protocol/identity": "3"`. If node_modules are fine, it should work fine.

**Review by vplasencia — APPROVED** [review]

File: apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx (hunk 1 above)
Comment (cedoor) [reply apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx]: I've gone into deep a bit, `@^3` might be better.

**Pull request merged**